<a href="https://colab.research.google.com/github/ADRIANVM117/proyectos_propios/blob/main/Estadistica/estadistica_ejercicio2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Con objeto de estimar la cantidad media que gasta un cliente en una comida en un importante restaurante, se recogieron los datos de una muestra de 49 clientes. Suponga que la desviación estándar de la población es $5. 

- a) ¿Cuál es el margen de error para 90% de confianza?
- b) Si la media poblacional es $24.80, ¿cuál es el intervalo de confianza de 90% para la media poblacional?

Desviación estandar poblacional 

# $ \sigma = \sqrt{\frac{\sum_{i = 1} (x_{i} - \bar{x})^2}{n}}$

Error estandar de muestra poblacional. 

# $ \frac{\sigma}{\sqrt{n}} $

# Margen de error. 

- Se conoce como margen de error a la cantidad de error en el muestreo aleatorio contemplado para cualquier investigación, especialmente al momento de aplicar cuestionarios o encuestas.

# $ E = Z_{\frac{\alpha}{2}} * \frac{\sigma}{\sqrt{n}}$

In [ ]:
import numpy as np
from scipy import stats

n = 49 # muestra
desv_pobla =5 # desviación poblacional
NC = .90    # nivel de confianza 
n_colas = 2 # numero de colas 
alpha = (1 -NC )/ n_colas    # alfa 
z_score =  stats.norm.ppf(1- alpha)  # valor critico de z 
error_pob = desv_pobla/ np.sqrt(n)   # error poblacional 
ME = z_score * error_pob   # Margen de error

print('Alpha : \t{}'            .format(alpha))
print('Error poblacional : \t{}'.format(error_pob))
print('Z-score : \t{:.4f}'.format(z_score))

print('Margen de error : \t{:.4f}'.format(ME))

Alpha : 	0.025000000000000022
Error poblacional : 	0.7142857142857143
Z-score : 	1.9600
Margen de error : 	1.4000


El margen de error es de 1.4000 con un nivel de confianza de 90 %

# Si la media poblacional es $24.80, ¿cuál es el intervalo de confianza de 90% para la media poblacional?


$ \mu = 24.80 $

# $ \bar{x} \pm \frac{\sigma}{\sqrt{n}} * Z_{\frac{\alpha}{2}}  $

In [ ]:
# Si la media poblacional es $24.80, ¿cuál es el intervalo de confianza de 90% para la media poblacional?

x = 24.80 

li = x - (error_pob* z_score)
ls = x + (error_pob* z_score)


print('Limite inferior : \t{:.3f}'.format(li))
print('Limite Superior : \t{:.3f}'.format(ls))

Limite inferior : 	23.400
Limite Superior : 	26.200


con un nivel de confianza del 95%. La gente gasta dinero en un intervalo de 23.400 y 26.200 pesos.